pkg/apis/operator/v1alpha1/tektonpipeline_validation.go

```diff
@@ -56,12 +59,20 @@ func (p *PipelineProperties) validate(path string) (errs *apis.FieldError) {
 		errs = errs.Also(apis.ErrInvalidValue(p.EnableApiFields, fmt.Sprintf("%s.enable-api-fields", path)))
 	}
 
+	if p.DisableInlineSpec != "" {
+		val := strings.Split(p.DisableInlineSpec, ",")
+		for _, v := range val {
+			if !slices.Contains(validatePipelineInlineSpecDisable, v) {
+				errs = errs.Also(apis.ErrInvalidValue(p.DisableInlineSpec, fmt.Sprintf("%s.disable-inline-spec", path)))
```

Comment:
do we need `fmt` here

Reply:
It's fine. Either you can use Concat or Sprintf. We are using Sprintf here everywhere.